packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -278,9 +278,10 @@ export class AngularCompilerPlugin {
       );
     }
 
-    if (!this._JitMode && !this._compilerOptions.enableIvy) {
-      // Only attempt to use factories when AOT and not Ivy.
-      this._useFactories = true;
+    if (!this._JitMode) {
+      // Only attempt to use factories when AOT or Ivy with factory shims.
+      this._useFactories = !this._compilerOptions.enableIvy
```

Comment:
@filipesilva was this not included on purpose of there it was just an oversight?